backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -26,7 +26,7 @@ import { Transaction as dbTransaction } from '@entity/Transaction'
 import { TransactionLink as dbTransactionLink } from '@entity/TransactionLink'
 
 import { TransactionTypeId } from '@enum/TransactionTypeId'
-import { calculateBalance, isHexPublicKey } from '@/util/validate'
```

Comment:
can we delete the whole function `isHexPublicKey` in `@/util/validate` please if possible?